Classes/Private/Manager/Audio/OCTAudioEngine.h

```diff
@@ -9,17 +9,23 @@
 #import <Foundation/Foundation.h>
 #import "OCTToxAV.h"
 
-typedef NS_ENUM(NSInteger, OCTAudioScope) {
-    OCTInput,
-    OCTOutput,
-};
-
 @interface OCTAudioEngine : NSObject
 
 @property (weak, nonatomic) OCTToxAV *toxav;
 @property (nonatomic, assign) OCTToxFriendNumber friendNumber;
 
 /**
+ * YES to not send audio frames over to tox, otherwise No.
+ */
+@property (nonatomic, assign) BOOL enableMicrophone;
+
+/**
+ * Default initializer.
+ * @param error Pointer to error object.
+ */
+- (instancetype)init:(NSError **)error;
```

Comment:
Yes. You can say in documentation that `setup...` method **has** to be called to make things work. :-) Also in this case there is no need in public `init` method.